apps/accounts/migrations/0083_carbontxtmotivation_providercarbontxt_and_more.py

```diff
@@ -0,0 +1,185 @@
+                (
+                    "created_by",
+                    models.ForeignKey(
+                        null=True,
+                        on_delete=django.db.models.deletion.SET_NULL,
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+                (
+                    "provider",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="carbon_txt",
+                        to="accounts.hostingprovider",
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+        ),
+        migrations.CreateModel(
+            name="ProviderCarbonTxtMotivation",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "content_object",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="accounts.hostingprovider",
+                    ),
+                ),
+                (
+                    "tag",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        related_name="%(app_label)s_%(class)s_items",
+                        to="accounts.carbontxtmotivation",
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+            },
+        ),
+        migrations.AddField(
+            model_name="hostingprovider",
+            name="carbon_txt_motivations",
+            field=taggit.managers.TaggableManager(
+                blank=True,
+                help_text="A comma-separated list of tags.",
+                through="accounts.ProviderCarbonTxtMotivation",
+                to="accounts.CarbonTxtMotivation",
+                verbose_name="Motivations for using carbon.txt",
+            ),
+        ),
+        migrations.RunSQL(
+            """
+
+                INSERT INTO accounts_providercarbontxt
+                (created, modified, domain, created_by_id, provider_id, is_delegation_set, carbon_txt_url)
+                SELECT
+                    created, modified, domain, created_by_id, provider_id,
+                    TRUE as is_delegation_set,
+                    CONCAT('https://', domain, '/carbon.txt') as carbon_txt_url
+                FROM accounts_linkeddomain
+                WHERE is_primary IS TRUE
+
+            """,
+        ),
+        migrations.RunPython(set_carbon_txt_urls),
+        migrations.DeleteModel(
```

Comment:
Move this to a later migration in a PR when we've validated everything works (along with changes to the models)